common/model/tests/test_transaction.c

```diff
@@ -49,12 +49,94 @@ void test_reset(void) {
   transaction_free(transaction);
 }
 
+void test_read_transaction_obj(void) {
+  int64_t test_value = 123456;
+  uint64_t test_timestamp = 1477037811737;
+  uint64_t test_attachment_timestamp = 1477037821737;
+  int64_t test_current_index = 2;
+  int64_t test_last_index = 5;
+  uint64_t test_attachment_timestamp_lower = 1477037831737;
+  uint64_t test_attachment_timestamp_upper = 1477037841737;
+  uint64_t test_snapshot_index = 2345;
+  bool test_solid = true;
+  uint64_t test_arrival_timestamp = 1477037812737;
+  iota_transaction_t *transaction = transaction_new();
+
+  // write transaction values
+  transaction_set_address(transaction, TEST_ADDRESS_0);
```

Comment:
These functions needs `flex_trit_t *`, not `tryte_t *`

Reply:
oh my fault